docs/maintainer/infrastructure.md

```diff
@@ -570,23 +665,22 @@ feedstocks get rerendered.
 
 For such ABI-compatible upgrades, similar but looser principles apply:
 
-- The pins are similarly defined in the global pinning, see [Globally Pinned Packages](pinning_deps.md#globally-pinned-packages).
 - We provide no support of any kind in terms of the long-term availability of a given compiler version.
 - We generally provide notice in the form of an announcement when a compiler is going to be upgraded.
 - Without promising any timelines, our compilers on Linux and OSX are normally
   very recent; on Windows, we generally use the last supported VS version.
 
-Despite the lack of explicit support, we try to keep the compilers in their various versions
-working also outside of conda-forge, and even provide an easy way to install them
-(through the [compilers feedstock](https://github.com/conda-forge/compilers-feedstock)).
+### More on compiler feedstocks
 
-More specifically, each compiler uses an _activation_ package that makes the difference
-between it being merely present in a build environment, and it being used by default.
-These will be installed when using `{{ compiler('xyz') }}` in `meta.yaml`, where
-`'xyz'` is one of `'c', 'cxx', 'fortran', 'cuda', 'rust', 'go-cgo', 'go-nocgo'`.
+The compiler activation and implementation packages are built by two separate
+feedstocks, with a few exceptions. The activation packages install scripts
+into `/etc/conda/activate.d` and `/etc/conda/deactivate.d` directories. These
```

Comment:
", which are invoked automatically by the conda build tool when those environments are activated"